GVRf/Framework/src/org/gearvrf/GVRScene.java

```diff
@@ -212,8 +213,21 @@ void updateStats() {
 
             mStatsConsole.writeLine("Draw Calls: %d", numberDrawCalls);
             mStatsConsole.writeLine(" Triangles: %d", numberTriangles);
+            
+            if(mStatMessage!="")
```

Comment:
Probably an TextUtils.isEmpty(mStatMessage) here is better and more safe than mStatMessage != "" check here.